makefile

```diff
@@ -439,6 +440,12 @@ tests-e2e-setup: setup-cluster
 		-e ARMADA_APICONNECTION_ARMADAURL="server:50051" \
 		-e ARMADA_APICONNECTION_FORCENOTLS=true \
 		armada-executor --config /e2e/setup/insecure-executor-config.yaml
+	docker run -d --name lookout-ingester-migrate  --network=kind -v ${PWD}/e2e:/e2e \
+		armada-lookout-ingester --config /e2e/setup/lookout-ingester-config.yaml --migrateDatabase
+	sleep 3
+	docker logs lookout-ingester-migrate
```

Comment:
Why do we need to get logs here?

Reply:
Are we able to check the exit code of the migration?  If  it's non-zero then we should bail here and print the logs.  If not, we can just assume the migration code worked and not print anything.